### PR TITLE
Add identification headers

### PR DESCRIPTION
**Pull request by mleko**

### CODE REVIEW

File: src/Header/IdentificationField.php [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+        $header = new static();
+        $header->setIds($messageIds);
+
+        return $header;
+    }
+
+    private static function trimMessageId($id)
+    {
+        return trim($id, "\t\n\r\0\xOB<>");
```
Comment (glensc) [inline src/Header/IdentificationField.php]: what is `\xOB` ? you meant `\x0B` or just `\v` ?

File: src/Header/InReplyTo.php [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+
+namespace Zend\Mail\Header;
```
Comment (glensc) [inline src/Header/InReplyTo.php]: i'm sure the copyright should be in every file

**Comment by Ocramius** [comment]
@Xerkus can you review this please?

**Comment by weierophinney** [comment]
Thanks, @mleko!

**Pull request merged**